docu/SICKAppSpaceCodingStarterKit_Documentation.md

```diff
@@ -21,8 +22,17 @@
       - [[ModuleName]_Processing.lua for multiple instances](#modulename_processinglua-for-multiple-instances)
     - [[ModuleName]_Controller.lua](#modulename_controllerlua)
       - [[ModuleName]_Controller.lua for multiple instances](#modulename_controllerlua-for-multiple-instances)
+	- [FlowConfig Functions and Events](#flowconfig-functions-and-events)
```

Comment:
Please check this line (and also the following ones) if capitalization is really correct here. Stay uniform within this table of content. Comparison: Line 33, here you write "functions and events"

Reply:
Done